code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -563,7 +563,8 @@
 /obj/item/slime_extract/cerulean/activate(mob/living/carbon/human/user, datum/species/jelly/luminescent/species, activation_type)
 	switch(activation_type)
 		if(SLIME_ACTIVATE_MINOR)
-			user.reagents.add_reagent(/datum/reagent/medicine/salbutamol,15)
+			ADD_TRAIT(user, TRAIT_NOBREATH, src)
```

Comment:
again, i dont think youre supposed to use refs for traits, use type?

Reply:
yes to this

Reply:
this one's actually okay since if it was a generic CERULEAN_TRAIT, then you wouldn't be able to stack them